benchmarks/spack/repo/packages/arepo/package.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from spack.package import *
+
+class Arepo(MakefilePackage):
+    """Arepo is a massively parallel gravity and hydrodynamics code for astrophysical simulations."""
+    homepage = "https://gitlab.mpcdf.mpg.de/vrs/arepo.git"
+    git = "https://gitlab.mpcdf.mpg.de/vrs/arepo.git"
+    
+    
+    version("master", branch="master")
+   
+
+    depends_on('mpi')
+    depends_on('gsl')
+    depends_on('hdf5')
```

Comment:
I had to add this dependency to build on my local machine
```suggestion
    depends_on('mpi')
    depends_on('gsl')
    depends_on('hdf5')
    depends_on('gmp')
```

Reply:
I have added the `depends_on('gmp')`